pkgs/stdenv/linux/default.nix

```diff
@@ -409,7 +409,7 @@ in
         # Simple executable tools
         concatMap (p: [ (getBin p) (getLib p) ]) [
             gzip bzip2 xz bash binutils.bintools coreutils diffutils findutils
-            gawk gnumake gnused gnutar gnugrep gnupatch patchelf ed
+            gawk gnumake gnused gnutar gnugrep gnupatch patchelf ed file
```

Comment:
Did you check that file doesn't reference any bootstrap files?

Reply:
Ah, no I did not.  Thank you for reminding me!

```
$ git log --oneline | head -n 2
dbdca8de867 libidn2: facilitate its self-autoreconf-ing behavior
0bec8faa22d fixLibtool(): patch ./configure, add `file` to common-path.nix
$ nix-store -q --requisites $(nix-build $(nix-instantiate . -A stdenv)) | grep bootstrap-tools
```

So, it looks like we're good.

Is there a reason why `pkgs/stdenv/linux/default.nix` doesn't use `disallowedRequisites` to make this check automatic?  For example, like #168590.  I suppose @Ericson2314 could answer this.